js/packages/openinference-instrumentation-llama-index/test/llamaIndex.test.ts

```diff
@@ -1,27 +1,30 @@
+import * as llamaindex from "llamaindex";
+
 import {
   InMemorySpanExporter,
   SimpleSpanProcessor,
 } from "@opentelemetry/sdk-trace-base";
 import { LlamaIndexInstrumentation, isPatched } from "../src/index";
 import { NodeTracerProvider } from "@opentelemetry/sdk-trace-node";
-import * as llamaindex from "llamaindex";
 import {
   SemanticConventions,
   OpenInferenceSpanKind,
   RETRIEVAL_DOCUMENTS,
 } from "@arizeai/openinference-semantic-conventions";
 
-const { Document, VectorStoreIndex } = llamaindex;
+const { Document, VectorStoreIndex, OpenAIEmbedding } = llamaindex;
 
 const DUMMY_RESPONSE = "lorem ipsum";
+// Mock out the embeddings response to size 1536 (ada-2)
+const FAKE_EMBEDDING = Array(1536).fill(0);
 
 const tracerProvider = new NodeTracerProvider();
 tracerProvider.register();
 
 const instrumentation = new LlamaIndexInstrumentation();
 instrumentation.disable();
 describe("llamaIndex", () => {
-  it("should pass", () => {
+  it("llamaindex should be available", () => {
     expect(true).toBe(true);
   });
```

Comment:
this was just a dummy test to make it so ci would pass when this package was empty, please remove!